### DISCUSSION TITLE
how can I save the data as archive to do another graphic type  on my computer?

### DISCUSSION THREAD

**Discussion by MatiasMarambio** [chemosim-lab/ProLIF#66] [Q&A]
When I run this command on Jupyter,

import seaborn as sns
import pandas as pd

# reorganize data
data = df.reset_index()
data = pd.melt(data, id_vars=["Frame"], var_name=["residue","interaction"])
data = data[data["value"] != False]
data.reset_index(inplace=True, drop=True)

# plot
sns.set_theme(font_scale=.8, style="white", context="talk")
g = sns.catplot(
    data=data, x="interaction", y="Frame", hue="interaction", col="residue",
    hue_order=["Hydrophobic", "HBDonor", "HBAcceptor", "PiStacking", "CationPi", "Cationic"],
    height=3, aspect=0.2, jitter=0, sharex=False, marker="_", s=8, linewidth=3.5,
)
g.set_titles("{col_name}")
g.set(xticks=[], ylim=(-.5, data.Frame.max()+1))
g.set_xticklabels([])
g.set_xlabels("")
g.fig.subplots_adjust(wspace=0)
g.add_legend()
g.despine(bottom=True)
for ax in g.axes.flat:
    ax.invert_yaxis()
    ax.set_title(ax.get_title(), pad=15, rotation=60, ha="center", va="baseline")

how can I save the data as archive to do another graphic type  on my computer?

I'm a newbie on Jupyter and ProLIF

**Comment by cbouy** [comment]
Hi,

You should just need to add this line at the end of the script after `ax.set_title(ax.get_title(), pad=15, rotation=60, ha="center", va="baseline")`:
```python
g.savefig('replace/this/with/the_path_to_your_output_image.png')
```

You can add [other arguments](https://matplotlib.org/stable/api/figure_api.html#matplotlib.figure.Figure.savefig) to customize the quality of the image generated.
